test/format/config.c

```diff
@@ -1050,11 +1050,13 @@ config_transaction(void)
             testutil_die(EINVAL, "prepare is incompatible with logging");
     }
 
-    /* Transaction timestamps are incompatible with implicit transactions. */
+    /* Transaction timestamps are incompatible with implicit transactions and logging. */
     if (GV(TRANSACTION_TIMESTAMPS) && config_explicit(NULL, "transaction.timestamps")) {
         if (GV(TRANSACTION_IMPLICIT) && config_explicit(NULL, "transaction.implicit"))
             testutil_die(
               EINVAL, "transaction.timestamps is incompatible with implicit transactions");
+        if (GV(LOGGING) && config_explicit(NULL, "logging"))
+            testutil_die(EINVAL, "transaction.timestamps is incompatible with logging");
```

Comment:
I agree, but it's a bigger change because the repeatable read and other snapshot isolation testing has to change, at least, I think that's the case. I opened WT-8793 for that change.